tests/utilities.py

```diff
@@ -0,0 +1,104 @@
+'''
+Module utilities, which provides objects for testing
+'''
+
+import pickle
+
+
+def anthology(list_of_paths_to_files_with_english_texts) -> str:
+    '''
+    Provides an anthology of English texts
```

Comment:
great that you comment your functions thoroughly

Reply:
@EfrainOlivaresUVA , I feel refreshed reading this as well. I have also experienced tension with bosses and coworkers when I have not understood code due to lack of comments and when I have taken time to write comments.